web3_proxy/src/stats/mod.rs

```diff
@@ -370,14 +371,16 @@ impl BufferedRpcQueryStats {
 
     async fn _compute_balance_deltas(
         &self,
+        sender_balance: balance::Model,
         referral_objects: Option<(referee::Model, referrer::Model)>,
     ) -> Web3ProxyResult<(Deltas, Option<(referee::Model, referrer::Model)>)> {
         // Calculate Balance Only
         let mut deltas = Deltas {
-            sender_available_balance_delta: -self.sum_credits_used,
-            sender_used_balance_delta: self.sum_credits_used,
+            balance_used_outside_free_tier: Default::default(),
+            balance_used_including_free_tier: Default::default(),
             sender_bonus_applied: false,
-            referrer_available_balance_delta: Decimal::from(0),
+            referrer_deposit_delta: Default::default(),
+            sender_bonus_balance_deposited: Default::default(),
         };
```

Comment:
nit: Deltas could derive(Default) and then this would just be `Deltas::default()`

Reply:
good point